src/psyclone/psyir/transformations/intrinsics/matmul2code_trans.py

```diff
@@ -71,10 +71,11 @@ def _get_array_bound(array, index):
         not supported.
 
     '''
+    from psyclone.psyir.nodes import DataNode
```

Comment:
To make pylint happy can you bring the import to the top or add a comment if there is a reason preventing it (circular dependency, ....)

Reply:
Yes that can be brought up. I also added pylint disables for the others